services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_utils.py

```diff
@@ -62,3 +72,36 @@ def all_containers_running(containers_inspect: List[DockerContainerInspect]) ->
     return len(containers_inspect) > 0 and all(
         (x.status == DockerStatus.RUNNING for x in containers_inspect)
     )
+
+
+async def cleanup_sidecar_stack_and_resources(
+    dynamic_sidecar_settings: DynamicSidecarSettings, scheduler_data: SchedulerData
+) -> None:
+    # remove the 2 services
+    await remove_dynamic_sidecar_stack(
+        node_uuid=scheduler_data.node_uuid,
+        dynamic_sidecar_settings=dynamic_sidecar_settings,
+    )
+    # remove network
+    await remove_dynamic_sidecar_network(scheduler_data.dynamic_sidecar_network_name)
+
+    # Remove all dy-sidecar associated volumes from node
+    unique_volume_names = [
```

Comment:
the volumes refer here are initially defined and mounted in ``docker_service_pecs.sidecar``

here those are explicitly repeated. How do you guarantee both definitions are in sync?